source/config/configFlags.py

```diff
@@ -6,7 +6,90 @@
 """Flags used for the configuration."""
```

Comment:
What more are you expecting?
Actually, the config flag is generic so there isn't much more to say with respect to feature flag here.
On the opposite, the feature flag is more specific, i.e. it's more or less a config flag with a default option. So maybe an update in the feature flag file? (even if we are a bit off-topic of this PR)

Just let me know.

Reply:
I want it to be clear to a developer that both modules exist, and that they have similar but different intentions. I think I'd suggest that feature flags should be used for new settings, since they provide a more intuitive way for a user to return to the default option.

Reply:
I have written up something in 9edee7e5ce957edea5665ac6d7a2849f0c4848fc.